romm/umbrel-app.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: romm_server_1
+      APP_PORT: 3003
+      PROXY_AUTH_ADD: "true"
+
+  server:
+    image: zurdi15/romm:1.10
```

Comment:
Hello @mateosilguero , I recommend using an image with its associated image digest. You can refer to this example here:

[Example Docker Compose File](https://github.com/getumbrel/umbrel-apps/blob/master/btc-rpc-explorer/docker-compose.yml#L10)

This is important because if the developer updates the application to the same tag, Umbrel may pull unverified changes upon restart. Including a digest addresses this concern.

To obtain the digest, execute: `docker pull zurdi15/romm:1.10`.

Additionally, ensure that the image is compatible with multiple CPU architectures (multi-architecture support).

Reply:
Check this out before implementing above changes:

https://github.com/getumbrel/umbrel-apps/pull/756#issuecomment-1733493232